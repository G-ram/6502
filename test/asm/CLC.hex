a9 00 18 
