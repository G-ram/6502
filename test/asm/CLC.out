A: 00
X: 00
Y: 00
P: 00010010
S: 00
PC: 603
100 : 00
101 : 00
